pkgs/top-level/python-packages.nix

```diff
@@ -2519,7 +2519,7 @@ in {
 
   google-auth-oauthlib = callPackage ../development/python-modules/google-auth-oauthlib { };
 
-  google-cloud-access-context-manager = callPackage ../development/python-modules/google-cloud-access-context-manager { };
+  google_cloud_access_context_manager = callPackage ../development/python-modules/google_cloud_access_context_manager { };
```

Comment:
You already changed a lot of these.... but 

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

Reply:
Going to remember that for future PRs.